components/tensorflow-notebook-image/Dockerfile

```diff
@@ -104,7 +104,7 @@ RUN pip3 --no-cache-dir install \
     jupyter-console==6.0.0 \
     jupyterlab \
     xgboost \
-    git+https://github.com/kubeflow/fairing@a9bb9d5cc1c9f1d75efa31198ddbdccfe176b7bf
+    kubeflow-fairing
```

Comment:
should we pin this to the latest version i.e. 0.7.1?

Reply:
That's a good question.

@abhi-g and @jinchihe, What are your preferences? 

1. In each Kubeflow release a Fairing maintainer could pin the version that they prefer to the release. Each Kubeflow release will require an update to this Dockerfile.
2. The notebook images could pick up the most recent version of Fairing at the time of release.  Depending on the order of operations, the most recent version of Fairing may not be included in the notebook image. However, no changes are required to this Dockerfile for each release.

Reply:
I prefer to pin to 0.7.1 because we know for now its tested with the rest of the KF components for training and serving. It does require some work to remember to update the version when releasing Fairing but thats trivial. 

If in the next few releases we get the indication that there is continuous activity and Fairing is keeping pace with rest of KF, we can choose to switch to install the latest release.

Reply:
Ok, I've added a commit to pin kubeflow-fairing to 0.7.1. I've built the image and tested it in KF 0.7.

PTAL